src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Functions/ProcessNextDataSetVersionMappingsFunction.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Functions;
 
-public class ProcessNextDataSetVersionFunction(
+public class ProcessNextDataSetVersionMappingsFunction(
```

Comment:
I think L16 and L17 below this need renaming from `ProcessNextDataSetVersion`  to `ProcessNextDataSetVersionMappings` to correspond with the renaming here and the renaming you've already done in the tests.

In the tests the inner class has been renamed to `ProcessNextDataSetVersionMappingsTests`.

Reply:
Cool yup thanks, renamed this method now